eng/Version.Details.xml

```diff
@@ -150,5 +150,29 @@
       <Uri>https://github.com/dotnet/runtime-assets</Uri>
       <Sha>4e8d82a2bdc31db2a3d392b4021b0c9ab19a4228</Sha>
     </Dependency>
+    <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Sdk" Version="6.0.1-alpha.1.20073.2">
```

Comment:
I don't think we need to list all platforms/architectures here, the `runtime.native.System.IO.Ports` on line 19 doesn't do it either. We might need to change how we're producing that package.

Reply:
`runtime.native.System.IO.Ports` is a metapackage, and we don't have a metapackage for LLVM. We can add one, sure

Reply:
Yeah I think we should do that.

Reply:
We should as the same package will be used for AOT tooling packages

Reply:
We're going to use one metapackage to pull in AOT support (i.e. `llc` and `opt` binaries) for every single supported architecture in one go?

Reply:
No, it will be RID driven for host (and possibly target) but the package should be identical to our internal dependency

Reply:
That's the split already in place here - `runtime.linux-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Sdk` contains LLVM for build-time (i.e. what we need internally to compile LLVM JIT Mono), `runtime.linux-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Tools` contains `opt` and `llc`, the two binaries needed for LLVM AOT support (i.e. end users would receive this, one way or another). If we want to rename things, I'm open to suggestions.